src/utils/process-get-filters.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+/**
+ * Removes the "filter[]" wrapper from a filter parameter
+ *
+ * @param {string} filterName
+ * @returns {string} the filter name with 'filter' and brackets removed
+ */
+const normalizeFilterName = (filterName) => {
+  const filterTextRegex = /filter\[(.*)\]/g;
+  const regexResults = filterTextRegex.exec(filterName);
+  return regexResults ? regexResults[1] : filterName;
+};
+
+/**
+ * Returns filters with keys changed to remove the "filter[]" wrapper
+ *
+ * @param {object} filters
+ * @returns {object}
+ */
+const normalizeFilterNames = (filters) => _.mapKeys(filters,
```

Comment:
Not really important, but I guess I'll make `normalizeFilterNames` and `normalizeFilterName` be static methods of `GetFilterProcessor` class.

Reply:
my goal was to make them private but I think you're right that having them be static is overall the better approach